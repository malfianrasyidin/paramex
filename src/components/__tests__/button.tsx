import { render } from '@testing-library/react';
import Button from 'components/button';

describe('Button', () => {
  it('renders correctly', () => {
    const { container } = render(<Button label="example" outline block />);

    expect(container).toMatchSnapshot();
  });
});
