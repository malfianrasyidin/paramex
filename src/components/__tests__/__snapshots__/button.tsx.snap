// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<div>
  <button
    class="button btn btn-outline-primary btn-block"
    type="button"
  >
    example
  </button>
</div>
`;
